src/main/java/frc/robot/Robot.java

```diff
@@ -48,12 +48,14 @@
     @Override
     public void robotInit() {
         // Initialize OI here
-        Camera0 = CameraServer.getInstance().startAutomaticCapture(0);
-        Camera1 = CameraServer.getInstance().startAutomaticCapture(1);
-        Camera0.setResolution(640, 480);
-        Camera1.setResolution(640, 480);
-        videoSink = CameraServer.getInstance().getServer();
+        camera0 = CameraServer.getInstance().startAutomaticCapture(0);
+        camera1 = CameraServer.getInstance().startAutomaticCapture(1);
+        camera0.setResolution(310, 240);
+        camera1.setResolution(310, 240);
+        camera0.setFPS(20);
+        camera1.setFPS(20);
         videoSink.getProperty("compression").set(70);
+        videoSink = CameraServer.getInstance().getServer();
```

Comment:
This line should be moved before the one above it - it's currently setting a property on something that doesn't exist.